docs/source/status.rst

```diff
@@ -0,0 +1,193 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+=====================
+Implementation Status
+=====================
+
+The following tables summarize the features available in the various official
+Arrow libraries.  Unless otherwise stated, the Python, R, Ruby and C/GLib
+libraries follow the C++ Arrow library.
+
+Data Types
+==========
+
++-------------------+-------+-------+-------+------------+-------+-------+
+| Data type         | C++   | Java  | Go    | JavaScript | C#    | Rust  |
+| (primitive)       |       |       |       |            |       |       |
++===================+=======+=======+=======+============+=======+=======+
+| Null              | ✓     | ✓     |       |            |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Boolean           | ✓     | ✓     | ✓     | ✓          |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Int8/16/32/64     | ✓     | ✓     | ✓     | ✓          |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| UInt8/16/32/64    | ✓     | ✓     | ✓     | ✓          |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Float16           |       |       |       |            |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Float32/64        | ✓     | ✓     | ✓     | ✓          |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Decimal128        | ✓     | ✓     |       |            |       |       |
```

Comment:
```suggestion
| Decimal128        | ✓     | ✓     | ✓     |            |       |       |
```

Reply:
Should this skip be removed?
https://github.com/apache/arrow/blob/master/dev/archery/archery/integration/datagen.py#L1502